py3status/cli.py

```diff
@@ -33,109 +33,108 @@ def parse_cli():
-    )
-    parser = argparse.ArgumentParser(add_help=True)
+    class HelpFormatter(argparse.HelpFormatter):
+        def _format_action_invocation(self, action):
+            metavar = self._format_args(action, action.dest.upper())
+            return "{} {}".format(", ".join(action.option_strings), metavar)
+
+    parser = argparse.ArgumentParser(formatter_class=HelpFormatter)
     parser.add_argument(
         "-b",
         "--dbus-notify",
         action="store_true",
         default=False,
         dest="dbus_notify",
-        help=(
-            "use notify-send to send user notifications "
-            "rather than i3-nagbar, "
-            "requires a notification daemon eg dunst"
-        ),
+        help="send notifications via dbus instead of i3-nagbar",
     )
     parser.add_argument(
         "-c",
         "--config",
+        metavar="FILE",
         action="store",
         dest="i3status_conf",
         type=str,
         default=i3status_config_file_default,
-        help="path to i3status config file",
+        help="configuration (default %(default)s)",
     )
     parser.add_argument(
-        "-d", "--debug", action="store_true", help="be verbose in syslog"
+        "-d",
+        "--debug",
+        action="store_true",
+        help="enable debug logging in syslog and --log-file",
     )
     parser.add_argument(
         "-g",
         "--gevent",
         action="store_true",
         default=False,
         dest="gevent",
-        help="enable gevent monkey patching (default False)",
+        help="enable gevent monkey patching",
     )
     parser.add_argument(
         "-i",
         "--include",
+        metavar="PATH",
         action="append",
         dest="include_paths",
-        help=(
-            "include user-written modules from those "
-            "directories (default ~/.i3/py3status)"
-        ),
+        help="append additional module paths",
     )
     parser.add_argument(
         "-l",
         "--log-file",
+        metavar="FILE",
         action="store",
         dest="log_file",
         type=str,
+        help="enable logging to FILE",
         default=None,
-        help="path to py3status log file",
+    )
+    parser.add_argument(
+        "-m",
+        "--disable-click_events",
+        action="store_false",
+        dest="click_events",
+        help="disable click events",
     )
     parser.add_argument(
         "-n",
         "--interval",
+        metavar="INT",
         action="store",
         dest="interval",
         type=float,
+        help="refresh interval for py3status (default %(default)s)",
         default=1,
-        help="update interval in seconds (default 1 sec)",
     )
     parser.add_argument(
-        "-s",
-        "--standalone",
-        action="store_true",
-        help="standalone mode, do not use i3status",
+        "-s", "--standalone", action="store_true", help="run py3status without i3status"
```

Comment:
I'd prefer this the way it was so that all these are the same format.

Reply:
I like the old way too. This is from `black`. It sometimes fix or break things.